### DISCUSSION TITLE
package-lock.json "modified"

### DISCUSSION THREAD

**Discussion by ghost** [FlareSolverr/FlareSolverr#454] [Q&A]
This is the only project that I have cloned where after I have installed and run the software I get a modified file notification for the **package-lock.json** file

Of course I can not push this change because I don't own the repo. I am new to Git so not sure if this is normal or an issue with how the software is deployed.

This is a freshly cloned repo.

<img width="1072" alt="Screen Shot 2022-08-02 at 6 00 04 PM" src="https://user-images.githubusercontent.com/9438406/182487728-297c6212-6281-41a0-bbf8-cd7655ef0648.png">

**Comment by ngosang** [comment]
I forget to update that file but it's not important.
